autopilot-daemon/pkg/handlers/healthchecks.go

```diff
@@ -30,7 +31,10 @@ func IntrusiveCheckTimer() {
 	utils.HealthcheckLock.Lock()
 	defer utils.HealthcheckLock.Unlock()
 	if utils.GPUsAvailability() {
+		klog.Info("GPUs are free. Can run invasive health checks.")
 		utils.CreateJob("dcgm")
```

Comment:
This print is a replication.. I added already these Info somewhere else in the code, so the same message would be printed twice with different wordings.
An example from a running pod
```
I0422 14:17:42.460243       9 healthchecks.go:30] Trying to run an intrusive check
I0422 14:17:42.640205       9 functions.go:61] Pod ap-ch-cfgfix-2c1-fz1-5e7-p10-v1-worker-11 with requests 8 and limits 8. Cannot run invasive health checks.
```

Reply:
Also, `utils.GPUsAvailability()` has been simplified and moved into `resources.go`

Reply:
All good as long as they're not duplicated. I'm not obsessed with where the prints are, as long as there are not too many saying the same thing